src/css/main.css

```diff
@@ -0,0 +1,229 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;600&display=swap');
+
+:root {
+  --black: #002;
+  --white: #fff;
+  --light-gray: #d3d3d3;
+  --dark-gray: #f3f0f0;
+}
+
+html {
+  overflow-x: hidden;
+}
+
+body {
+  position: relative;
+  overflow-x: hidden;
+}
+
+.nav-wrapper {
+  border: 2px solid var(--black);
+  padding: 8px;
+  border-radius: 8px;
+  margin: 10px auto;
+  width: 95%;
+}
+
+.logo {
+  font-weight: 700;
+  color: var(--black);
+}
+
+.navbar {
+  justify-content: space-between;
+  width: 90%;
```

Comment:
I think this width should be set to 100% and padding should be added to the parent element to achieve what you want to achieve here.